curriculum/challenges/english/16-the-odin-project/top-learn-variables-and-operators/learn-variables-and-operators-lesson-i.md

```diff
@@ -83,18 +83,23 @@ console.log( counter++ ); // 0
 
 ## --text--
 
-What is the difference in the result of the two console log statements in the JavaScript code below?
-
+What are the outputs of the two `console.log` statements in the JavaScript code below?
 ```js
 let counter = 1;
 console.log(2 * ++counter); // Statement A
+```
 
+```js
 let counter = 1;
 console.log(2 * counter++); // Statement B
 ```
 
 ## --answers--
 
+There is no difference; both `console.log` statements will show the same result.
+
+## --answers--
+
 There is no difference; both alert statements will show the same result.
```

Comment:
you have `## --answers--` twice, and bot the new answer and the old answer. Please remove the unnecessary lines.